contrib/earthfile-syntax-highlighting/syntaxes/earthfile.tmLanguage.json

```diff
@@ -46,6 +46,14 @@
 				{
 					"name": "comment.line.earthfile",
 					"match": "(\\s+)((#).*$\\n?)"
+				},
+				{
+					"captures": {
+						"2": {
+							"name": "comment.line.number-sign.earthfile"
+						}
+					},
+					"match": "([\\\\\\s]+)((#).*$\\n?)"
```

Comment:
Need to allow whitespace plus the line continuation. This turns into an escape explosion.

Reply:
PLus use the capture group so we dont count the slash as part of the comment :( if only I could use lookahead/lookbehind